Crestone-Tech/ctechsite
# CT-58-pass-projectInfo-to-ProjectCard

## Purpose

This is the last step in the following
- define TS data type for Project details (name, description, image URL, etc)
- create data file with two projects defined
- Projects component gets the data and uses map() to add a ProjectCard per project
- Project Card displays/uses data from the data file

## Fixes
 https://crestonetech.atlassian.net/browse/CT-58

## Steps to Reproduce or Test
1) look at data file ./data/projectsData.ts
2) open site
3) observe that the projects name, description and image URL match those in the data file
